packages/react-composites/src/composites/CallComposite/components/HoldPane.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PrimaryButton, Stack, Text, useTheme } from '@fluentui/react';
+import { _pxToRem } from '@internal/acs-ui-common';
+import React, { useState } from 'react';
+import { useLocale } from '../../localization';
+import {
+  holdPaneContentStyles,
+  holdPaneLabelStyles,
+  holdPaneTimerStyles,
+  paneStyles,
+  resumeButtonStyles
+} from '../styles/HoldPane.styles';
+
+/**
+ *
+ */
+export interface HoldPaneProps {
+  onToggleHold: () => Promise<void>;
+}
+
+/**
+ * Hold pane to display when the user places themselves on hold
+ *
+ * @beta
+ */
+export const HoldPane = (props: HoldPaneProps): JSX.Element => {
+  const { onToggleHold } = props;
+  const locale = useLocale();
+  const theme = useTheme();
+
+  const [rawTime, setRawTime] = useState<number>(0);
+  const [time, setTime] = useState<string>('00:00');
+
+  React.useEffect(() => {
+    const interval = setInterval(() => {
+      setRawTime((time) => time + 10);
+
+      const mins: string =
```

Comment:
nit: to make it easy to read this block, i'd recommend pulling out these helpers to const functions to the bottom of this file:
```
const getMinutes = (duration: number): string =>  Math.floor....
const getSeconds = (duration: number): string =>  Math.floor....
```